src/common/sockman.h

```diff
@@ -39,6 +44,12 @@ class SockMan
         STOP_LISTENING,
     };
 
+    /**
+     * Constructor.
+     * @param[in] interrupt_net Singal this to tell SockMan to cease network activity.
```

Comment:
In commit "net: move I2P-accept-incoming code from CConnman to SockMan" (3e91c8b502d43d656d9f3207cb550a0c03d1429e)

s/Singal/Signal